schematic/manifest/commands.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env python3
 
+import warnings
```

Comment:
Why was this added? I don't see this module being used anywhere in this file.

Reply:
Thanks, removing `warnings`.